site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -561,26 +559,28 @@ A reporting mechanism for losing bidders is [under discussion](https://github.co
 {% endDetails %}
 
 
-{: #trusted-server-detail}
+{: #key-value-service-detail}
 
 {% Details %}
 
 {% DetailsSummary %}
-## What is a trusted server?
+## What is a FLEDGE Key/Value service?
 
 {% endDetailsSummary %}
 
-In the context of the Privacy Sandbox, a trusted server is a secure environment to enable access to
-data, while preserving privacy. A request to a trusted server cannot result in event-level logging,
-or have other side effects.
+FLEDGE Key/Value service allows adtechs to query for realtime data when a bid is made by the buyer or when an ad is scored by the seller while preserving privacy. FLEDGE Key/Value service is one of the [FLEDGE services](/blog/fledge-service-overview/). 
+
+The Key/Value service is deployed to the adtech's own cloud infrastructure, and the service runs on a [trusted execution environment](#trusted-execution-environment). A request to a Key/Value service cannot result in event-level logging or have other side effects. The Key/Value service will also support [user-defined functions (UDFs)](https://github.com/WICG/turtledove/blob/main/FLEDGE_Key_Value_Server_trust_model.md#support-for-user-defined-functions-udfs) that allows adtechs to execute their own custom logic within the Key/Value service. 
+
+{: #key-value}
+
+When a buyer or seller makes a request to a FLEDGE Key/Value service, they specify the data they require from the server as a list of 'keys'. The Key/Value service responds with a value for each key.
```

Comment:
Maybe "A buyer or seller provides a list of 'keys' to specify the data they require from a FLEDGE Key/Value service ."